### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by schmunk42**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #16  #17

### CODE REVIEW

**Comment by schmunk42** [comment]
Push to Docker Hub needs to be done after merge.

**Comment by samdark** [comment]
> Push to Docker Hub needs to be done after merge.

How?

**Comment by schmunk42** [comment]
> How?

The push step(s) actually are only running on `master`, we need to set the variables `DOCKER_...` (but I have no access).
But this can also done before.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 </p>
 
 **Stable**
-[![Build Status](https://travis-ci.com/yiisoft/yii-docker.svg?branch=master)](https://travis-ci.com/yiisoft/yii-docker)
+tbd
```
Comment (rustamwin) [inline README.md]: What does it mean?

File: php/Dockerfile-debian [hunk 2]
```diff
@@ -134,8 +60,10 @@ RUN if command -v a2enmod >/dev/null 2>&1; then \
     ;fi
 
 # Install Yii framework bash autocompletion
-RUN curl -L https://raw.githubusercontent.com/yiisoft/yii2/master/contrib/completion/bash/yii \
-        -o /etc/bash_completion.d/yii
+RUN apt-get install bash-completion && \
+    mkdir /etc/bash_completion.d && \
+    curl -L https://raw.githubusercontent.com/yiisoft/yii2/master/contrib/completion/bash/yii \
```
Comment (rustamwin) [inline php/Dockerfile-debian]: Doesn't seem to work with yii3

File: README.md (hunk 1 above)
Comment (schmunk42) [reply README.md]: to be done

when build on GH passes

File: php/Dockerfile-debian (hunk 2 above)
Comment (schmunk42) [reply php/Dockerfile-debian]: Is there an auto-completion for yii3?
Otherwise I'd remove this.

Comment (samdark) [reply php/Dockerfile-debian]: Nope, there isn't.

**Comment by samdark** [comment]
@schmunk42 I've fixed repo permissions. Now you should have access to environment variables.

**Comment by schmunk42** [comment]
> @schmunk42 I've fixed repo permissions. Now you should have access to environment variables.

@samdark I removed the bash completion and added the secrets.

**Comment by samdark** [comment]
Merge time?

File: README.md (hunk 1 above)
Comment (samdark) [reply README.md]: Could be applied now, I think.

Comment (schmunk42) [reply README.md]: added

**Comment by schmunk42** [comment]
Can be merged.

**Comment by samdark** [comment]
Thank you!

**Pull request merged**